src/Sarif.Multitool.FunctionalTests/Update-Expected.ps1

```diff
@@ -28,11 +28,12 @@ param(
 
 $ErrorActionPreference = "Stop"
 
+Import-Module -Force ..\..\scripts\Projects.psm1
 Import-Module -Force ..\..\scripts\ScriptUtilities.psm1
 
 $ExpectedSuffix = "_Expected$SarifExtension"
 
-$multiToolExePath = "$BinRoot\Sarif.Multitool\AnyCPU_$Configuration\net461\Sarif.Multitool.exe"
+$multiToolExePath = "$BinRoot\${Platform}_$Configuration\Sarif.Multitool\$($Frameworks.NetFx[0])\Sarif.Multitool.exe"
```

Comment:
>$($Frameworks.NetFx[0]) [](start = 73, length = 23)

Avoid hard-coding the .NET Framework version we build for. The ideal is that it occurs only in one place, in build.common.props.